cmake/genai_flatbuffers.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+file(GLOB genai_flatbuffers_srcs CONFIGURE_DEPENDS
+    "${CMAKE_SOURCE_DIR}/src/flatbuffers/*.h"
+    "${CMAKE_SOURCE_DIR}/src/flatbuffers/*.cc"
+    )
+
+add_library(genai_flatbuffers STATIC ${genai_flatbuffers_srcs})
+target_link_libraries(genai_flatbuffers PRIVATE FlatBuffers::FlatBuffers)
```

Comment:
Make this PUBLIC so the other targets do not need to link against FlatBuffers as well.